test/fixtures/__shared__/util/scripts.js

```diff
@@ -33,9 +33,9 @@ function execaSafe(...args) {
       stderr: stripYarn(
         stripAnsi(
           err.message
-            .split(os.EOL)
+            .split('\n')
```

Comment:
We're trying to add back Windows support here. Do we really want to change this?

Reply:
@willsmythe do you remember why this was changed?

Reply:
The line separator for console messages on Windows is `\n`. Not sure if this is always the case, or just with the way yarn is being invoked in these tests. I think if you undo this change, you'll see certain tests starting to fail on Windows because os.EOL will be `\r\n`